cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -171,6 +177,27 @@ logExceptContinue renderError f = do
       return Nothing
     Right a -> return (Just a)
 
+-- | Calculate the percentage sync rendered as text.
+percentage
+  :: RelativeTime
+  -- ^ 'tolerance'.  If 'b' - 'a' < 'tolerance', then 100% is reported.  This even if we are 'tolerance' seconds
+  -- behind, we are still considered fully synced.
+  -> RelativeTime
+  -- ^ 'nowTime'.  The time of the most recently synced block.
+  -> RelativeTime
+  -- ^ 'tipTime'.  The time of the tip of the block chain to which we need to sync.
+  -> Text
+percentage tolerance a b = Text.pack (printf "%.2f" (fromIntegral @Int @Double (ua * 10000 `div` ub) / 100.0))
+  where st = relativeTimeSeconds tolerance
+        -- Plus 1 to prevent division by zero
+        sa = relativeTimeSeconds a + 1
```

Comment:
What is the s in `sb` and `sa` ?

Reply:
I added more comments.  It originally means **S**econds, but I now decide it will mean "strictly positive" (ie non-zero)